umi_tools/group.py

```diff
@@ -403,7 +440,7 @@ def main(argv=None):
                         else:
                             # Add the 'UG' tag to the read
                             read.tags += [('UG', unique_id)]
-                            read.tags += [('FU', top_umi)]
+                            read.tags += [('BX', top_umi)]
```

Comment:
This is inline with Picard Markduplicates with molecular barcodes which uses RX for raw UMIs and BX for "corrected UMIs" - It's not clear how they 'correct' UMIs but the release notes for 2.7.0 says 'UMI Aware Mark Duplicates with rudimentary error correction' 
https://github.com/broadinstitute/picard/releases